components/FoundItem.js

```diff
@@ -1,25 +1,28 @@
 import React from "react";
 import Image from "next/image";
 import styled from "styled-components";
+import Link from "next/link";
 
 export default function FoundItem({ name, description, img, id }) {
   return (
-    <div>
-      <li>
-        This item is: {name}. Its description is: {description}. Its ID is: {id}
-        . It looks like:
-        <ImageContainer>
-          <Image
-            objectFit="cover"
-            src={img}
-            width={200}
-            heigth={200}
-            layout="fill"
-            alt="lost items"
-          />
-        </ImageContainer>
-      </li>
-    </div>
+    <>
+      <Link href={`/details/${id}`}>
```

Comment:
I am not quite sure if this is the right spot for the <Link />. Because now you could click like everywhere within the fragment and get to the details page. I would suggest to put the Link around the Image (would make more sense to me at the moment)